src/main/java/com/faforever/client/vault/replay/ReplayVaultController.java

```diff
@@ -120,6 +124,25 @@ protected void loadLocalReplaysInBackground() {
   @NotNull
   private TableRow<Replay> replayRowFactory() {
     TableRow<Replay> row = new TableRow<>();
+    ContextMenu contextMenu = new ContextMenu();
+    MenuItem copyReplayId = new MenuItem("Copy Replay Id");
+    copyReplayId.setOnAction(event -> {
+      var replayId = row.getItem().getId();
+      final Clipboard clipboard = Clipboard.getSystemClipboard();
+      final ClipboardContent content = new ClipboardContent();
+      content.putString(String.valueOf(replayId));
+      clipboard.setContent(content);
+    });
+    MenuItem copyReplayPath = new MenuItem("Copy Replay Path");
+    copyReplayPath.setOnAction(event -> {
+      var replayPath = row.getItem().getReplayFile();
```

Comment:
We avoid the var keyword for code style reasons

Reply:
changed this, thanks